verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -63,7 +63,27 @@ export default class CasesController {
     };
 
     create = async (req: Request, res: Response): Promise<void> => {
-        // TODO: Geocode if no lat/lng provided.
+        // Geocode query if no lat lng were provided.
```

Comment:
What if there is a lat/lng but no country, admin1, etc? Would we want to geocode to get those values from the lat/lng?

Reply:
There's no reverse geocoding yet, I could add that but the current geocoder only does forward geocoding.